packages/gestalt/src/IconButton.tsx

```diff
@@ -235,18 +260,31 @@ const IconButtonWithForwardRef = forwardRef<HTMLButtonElement, Props>(function I
     </button>
   );
 
-  return tooltip?.text ? (
-    <Tooltip
-      accessibilityLabel={tooltip.accessibilityLabel}
-      idealDirection={tooltip.idealDirection}
-      inline={tooltip.inline}
-      text={tooltip.text}
-      zIndex={tooltip.zIndex}
-    >
-      {buttonComponent}
-    </Tooltip>
-  ) : (
-    buttonComponent
+  const labelComponent = (
+    <div className={labelStyle}>
+      <Text align="center" color={labelColor} lineClamp={2} size="100" weight="bold">
```

Comment:
https://github.com/pinterest/gestalt/pull/3720#issuecomment-2313609266. Did we implement this?

Reply:
No, we decided the behavior should be the same as Button, the text is just external here.
So, like Button, the only accessibility label read is the accessibilityLabel string set.